app/validators/status_prerequisite_validator.rb

```diff
@@ -46,7 +49,8 @@ def can_be_2i_approved?(record)
       (
        record.status_was == "in_review" ||
        record.status_was == "scheduled" ||
-       record.status_was == "published"
+       record.status_was == "published" ||
+       record.status_was == "approved_2i"
```

Comment:
This could now be refactored to something like:

```ruby
allowed_previous_statuses = %w(in_review scheduled published approved_2i)
allowed_previous_statuses.include? record.status_was
```